integration_tests/test_const_list.py

```diff
@@ -1,4 +1,4 @@
-from lpython import i32, list, str, Const
+from lpython import i32, Const
```

Comment:
`list` and `str` are default types provided by `CPython`. They are not defined in the `lpython` module.